video/opengl.h

```diff
@@ -57,9 +57,9 @@ class OpenGL : public Input
         return false;
     }
 
-private:
+protected:
```

Comment:
Maybe add public getters rather than doing this? Protected member baaaad, public getters better (or something)

Reply:
That probably would be clearer...